internal/pgadmin/config.go

```diff
@@ -40,6 +40,10 @@ const (
 	dataVolume    = "pgadmin-data"
 	dataMountPath = "/var/lib/pgadmin"
 
+	// ldapPasswordMountPath is the path for mounting the LDAP Bind Password
+	ldapPasswordPath         = "~postgres-operator/ldap-bind-password" /* #nosec */
```

Comment:
⛏️ Comment is `ldapPasswordMountPath` while variable is `ldapPasswordPath`.

Reply:
:+1: updated.